src/Data/String/WrappedLine.purs

```diff
@@ -0,0 +1,96 @@
+{-
+Copyright 2016 SlamData, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-}
+
+module Data.String.WrappedLine (MeasuredWord, WrappedLine, Conf, measuredWords, wrappedLines, printWrappedLine, printWord) where
```

Comment:
Is hiding the constructors here intentional? If so, we should remove the `newtype` instance for `MeasuredWord` - it's kinda a bug that it lets you have that instance with hidden constructors at the moment.